### ISSUE TITLE
yahoo.finance.incomestatement and balancesheet tables YQL queries are not working

### ISSUE THREAD

**Issue by ghost** [yql/yql-tables#473]
These calls were working earlier but now they have stopped working. Any idea when this could be fixed?

https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20yahoo.finance.balancesheet%20where%20symbol%20%3D%20%27T%27&format=json&diagnostics=true&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys&callback=

INcome Statement : 
https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20yahoo.finance.incomestatement%20where%20symbol%20%3D%20%27T%27&format=json&diagnostics=true&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys&callback=

Sample output:-
{"query":{"count":1,"created":"2016-04-24T13:15:29Z","lang":"en-US","diagnostics":{"url":[{"execution-start-time":"0","execution-stop-time":"2","execution-time":"2","content":"http://www.datatables.org/yahoo/finance/yahoo.finance.incomestatement.xml"},{"execution-start-time":"4","execution-stop-time":"28","execution-time":"24","content":"http://finance.yahoo.com/q/is?s=T&quarterly"}],"publiclyCallable":"true","javascript":{"execution-start-time":"3","execution-stop-time":"30","execution-time":"26","instructions-used":"5014","table-name":"yahoo.finance.incomestatement"},"user-time":"31","service-time":"18","build-version":"0.2.438"},"results":{"incomestatement":{"symbol":"T","timeframe":"quarterly"}}}}

In short, the data is missing in the above calls. Can this be fixed please? A lot of people are looking for help on the yahoo finance table issues.

**Comment by hacksut3l** [comment]
Don't you have updates on this one?

**Comment by saurabhgohil11** [comment]
Any update on this issue?

**Comment by ghost** [comment]
Nope, not yet. I am waiting from Yahoo to respond onto this one.

Thanks & Regards,
Jitesh Dundas

Phone - 732-357-6292
